backend/middleware/isRedirectActive.ts

```diff
@@ -0,0 +1,9 @@
+import { isRedirectActive, isRedirectCaseLoad, isRedirectEnabled } from '../utils'
+
+export default () => async (req, res, next) => {
+  req.session.isRedirectActive = isRedirectActive(
+    isRedirectEnabled(res),
+    isRedirectCaseLoad(req?.session?.userDetails?.activeCaseLoadId)
+  )
```

Comment:
If you put the value into res.locals.isRedirectActive rather than on the session then it will be available in the view without the need to return it in a res.render. That was how I thought you could remove from business logic of the controller.

It may be over the top, I wasn't sure on how much this redirect would be used.